definitions.go

```diff
@@ -201,13 +201,13 @@ type ComponentExportRepresentation struct {
 
 // ComponentRepresentation struct
 type ComponentRepresentation struct {
-	Config       *MultivaluedHashMap `json:"config,omitempty"`
-	ID           *string             `json:"id,omitempty"`
-	Name         *string             `json:"name,omitempty"`
-	ParentID     *string             `json:"parentId,omitempty"`
-	ProviderID   *string             `json:"providerId,omitempty"`
-	ProviderType *string             `json:"providerType,omitempty"`
-	SubType      *string             `json:"subType,omitempty"`
+	Config       *map[string][]string `json:"config,omitempty"`
```

Comment:
Don't use pointer for map and slices: they already are nilable... We used to do that in our first versions but it only causes problems, no benefits.